native/x64/version.txt

```diff
@@ -0,0 +1,2 @@
+Scintilla: 5.1.5
+Lexilla: 5.1.4
```

Comment:
This also has different version:
![image](https://user-images.githubusercontent.com/40712699/213359538-856ec7ad-7385-4004-b23e-833c9bb4bda7.png)